### DISCUSSION TITLE
Hybrid Mode Discussion

### DISCUSSION THREAD

**Discussion by immdevrov** [vuejs/language-tools#4659] [General]
Hi! Thanks for all your work.
I want to re-open discussion that took place [here](https://github.com/vuejs/language-tools/pull/4119#issuecomment-2014596799)

What differences between hybridMode=true and hybridMode=false? Is something better? Is hybrid mode works better with up-to-date typescript (or vice versa). What is the reason to use one way or another?

**Comment by so1ve** [comment]
If I understood correctly, using hybridMode will make other ts plugins work in `.vue` files? If hybridMode is disabled then some vscode extensions, such as styled-components won't work

**Comment by immdevrov** [comment]
Well, we are talking about volar in general, not only vscode extension? Or this make sense only for vscode? Then, if I am not using vscode so there is no practical difference?

**Comment by so1ve** [comment]
Unfortunately I'm not familiar about volar.js 🫠

**Comment by vidal7** [comment]
Maybe this can help : https://gist.github.com/johnsoncodehk/62580d04cb86e576e0e8d6bf1cb44e73
